models/Asset/Image/Thumbnail.php

```diff
@@ -162,7 +164,14 @@ public function __toString(): string
     private function addCacheBuster(string $path, array $options, Asset $asset): string
     {
         if (isset($options['cacheBuster']) && $options['cacheBuster']) {
-            if (!str_starts_with($path, 'http')) {
+             if (
+            isset($options['cacheBuster']) && 
+            $options['cacheBuster'] && 
+            !str_starts_with($path, 'http') && 
+            !str_starts_with($path, '/cache-buster-')
```

Comment:
Sorry maybe I should have mentioned that the suggest does not really work for that :)
It was just meant as an example, not really something to commit.

Reply:
Feel free to cleanup the code as you wish. I just sticked to the codebase as it was.